scripts/core/exp-usm-p2p.yml

```diff
@@ -0,0 +1,113 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+# See LICENSE.TXT
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Unified Runtime Experimental APIs for Usm P2P"
+ordinal: "99"
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Supported peer info"
+class: $xUsmP2P
+name: $x_exp_peer_info_t
+typed_etors: True
+etors:
+  - name: PI_PEER_ACCESS_SUPPORTED
```

Comment:
If `PI` refers to Plugin Interface we should remove this reference.

Reply:
Thanks, I've corrected the naming.